backend/scripts/seed.sh

```
@@ -0,0 +1,1312 @@
+        ]}" >/dev/null
+    ok "Polls + votes/rankings created"
+
+    # Pitches – one from each user
+    local p1 p2 p3
+    p1=$(insert_pitch_sql "$trip_id" "$my_id" \
+        "The case for Bali in December" \
+        "Dry season runs until October but December is actually still great. Rainy season means lush green landscapes, fewer tourists, and significantly cheaper flights and accommodation. I did the math and we save roughly 40% vs. peak season." 105)
+    p2=$(insert_pitch_sql "$trip_id" "$MAYA_ID" \
+        "I found the perfect villa" \
+        "There is a 5-bedroom compound in Ubud with private pool and staff. Split 5 ways it is less than a mid-range hotel. I will drop the Airbnb link in chat." 70)
+    p3=$(insert_pitch_sql "$trip_id" "$CARLOS_ID" \
```

Shellcheck (0.11.0)

[warning] p3 appears unused. Verify use (or export if used externally).

(SC2034)